crates/aranya-client/src/afc.rs

```diff
@@ -0,0 +1,492 @@
+//! AFC support.
+use std::{fmt::Debug, sync::Arc};
+
+use anyhow::Context;
+use aranya_daemon_api::{self as api, AfcShmInfo, ChanOp, DaemonApiClient, CS};
+use aranya_fast_channels::{
+    self as afc,
+    shm::{Flag, Mode, ReadState},
+    Client as AfcClient,
+};
+use serde::{Deserialize, Serialize};
+use tarpc::context;
+use tokio::sync::Mutex;
+use tracing::debug;
+
+use crate::{
+    error::{aranya_error, IpcError},
+    DeviceId, LabelId, Result, TeamId,
+};
+
+/// Locally unique AFC channel ID.
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Ord, PartialOrd, Hash, Serialize, Deserialize)]
+pub struct ChannelId(api::AfcChannelId);
+
+/// AFC sequence number identifying the position of a ciphertext in a channel.
+#[derive(Copy, Clone, Debug, Default, Hash, Eq, PartialEq, Ord, PartialOrd)]
+pub struct Seq(afc::Seq);
+
+impl Seq {
+    /// Convert AFC sequence object to `u64`.
```

Comment:
We don't need to specify that everything in this file is for AFC. Eg, say "Converts the sequence number to a `u64`."